src/main/utils/PlaybackAPI.js

```diff
@@ -134,6 +138,17 @@ class PlaybackAPI {
     this._save();
   }
 
+  _setPauseAfter(mode) {
```

Comment:
This method should not be in `PlaybackAPI`.  The playback API exposes information about the player to an external controller, through either the JSON api or the WebSocket API.  This appears to be a method that controls the player so should not really be here.